docs/OnboardingOutcomes.md

```diff
@@ -0,0 +1,121 @@
+---
+resourceType: Documentation
+personas: 
+  - Developer
+  - Product Owner
+  - Designer
+tags:
+  - openshift
+  - new
+  - onboarding
+  - project set
+  - devops
+description: Learn where to ask for help with issues not covered by devops-requests
+---
+
+# Welcome to our Platform Community! 
+
+Glad you’re here! If you’ve already had your onboarding meeting with the Platform Services team, you’ll find this follow-up info helpful as part of your welcome package to our community. If you’re just scoping us out, we hope you like what you learn and will come join our journey.  
+
+## We’re Better Together
+
+Dreamers. Vanguards. Innovators. We’re all creating lasting change by moving our government into a brave new world of openness and agility. **We can’t do it alone and we don’t pretend we can.** So, we’re holding hands and setting off on an adventure together to go towards something greater.
+
+You’re here to explore a new, more digital government. You’re building the path that others will follow. You’re the leader that you always wish you had, and with the right people by your side, you’ll have the courage to keep going.
+
+Try. Experiment. Iterate. Fail. Try again. Change the world, together.
+
+## What this means for YOU
+
+In short, it means that the value you bring to our community is not what you do for yourself. **Your value is determined by what you do for others**. You get to work with a sense of purpose, cause and belief as you contribute to something even bigger than your own project. Not bad for a paid gig.
+
+### A Community-Supported Platform
+
+Our Platform is supported by our community. Quite simply, this means: we help each other. **Your first stop for help should always be the community**. This refers to both the Developer Exchange community, and the larger development community on the web.
+
+* Don’t know what to do? Say it out loud to our community and chances are that someone who does know will offer help.
+* Holding onto a great idea, or have a brilliant answer? Give it away, and watch it become even better.  
```

Comment:
* Holding onto a great idea, or have a brilliant answer? Contribute it.  Share it.  You'll be amazed at how quickly the community will join you in your journey